frontend/src/components/CloseSuspendReasonModal.css

```diff
@@ -0,0 +1,30 @@
+.smart-hub--goal-close-suspend-reason .usa-form-group fieldset textarea {
+    resize: none;
+    width: 415px;
```

Comment:
why turn off resize?

Reply:
Why not.

Reply:
In all seriousness, its fixed in the mockup.